### DISCUSSION TITLE
GitHub Pages deploy size limit

### DISCUSSION THREAD

**Discussion by cat-master21** [community/community#28089] [Other Feature Feedback, Questions, & Ideas] [answered]
I was unable to find the size limit for GitHub Pages. I want to know this because you can:
1. Use Actions to build the website
2. And use submodules.
This can take up much space and I would like to know the limit as it does not seem to be documented.

**Comment by zigad** [comment, marked as answer]
Maybe this can help?

```
Usage limits

GitHub Pages sites are subject to the following usage limits:

GitHub Pages source repositories have a recommended limit of 1 GB. For more information, see "[What is my disk quota?](https://docs.github.com/en/articles/what-is-my-disk-quota/#file-and-repository-size-limitations)"

Published GitHub Pages sites may be no larger than 1 GB.

GitHub Pages sites have a soft bandwidth limit of 100 GB per month.

GitHub Pages sites have a soft limit of 10 builds per hour. This limit does not apply if you build and publish your site with a custom GitHub Actions workflow
```


Source: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages

**Reply by cat-master21** [reply]
Apperently I missed it this section some how. Thanks

**Reply by zigad** [reply]
> Apperently I missed it this section some how. Thanks

No problem! That's why we are here!

Good luck!
